assembl/static2/js/app/components/debate/common/cancelTranslationForm.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
```

Comment:
I would have created a new folder: `components/debate/translations/cancelTranslations.jsx`

Reply:
ok, I see, put all stuff related to translations in a specific folder. I can do that

Reply:
exact!

Reply:
done : https://github.com/assembl/assembl/commit/4467c19543990f81ee74465cfa156f224704526e